libraries/AP_UAVCAN/dsdl/ardupilot/equipment/gnss/20002.Inject.uavcan

```diff
@@ -0,0 +1,8 @@
+#
+# GNSS Inject, used by autopilots to forward GPS_RTCM_DATA or GPS_INJECT mavlink msgs
+# to GNSS nodes. Normally sent as broadcast
+
+# flags follows the same conventions as MAVLink GPS_RTCM_DATA
+uint8 flags
```

Comment:
Lets do the flags correctly here. We live with the bad flags in MAVLink for compatibility reasons with PX4, but we really shouldn't continue it on.

Reply:
what form would you like the flags in? I thought part of the issue was that meanings of the bits wasn't in the XML. I could add in DSDL const values.
I do like being able to fwd the pkts direct to CAN, so I know that the way it behaves with a UAVCAN GPS is identical to the way it behaves on a UART. That is a huge win as we've had plenty of testing of the mavlink msg, so behaving the same way gives me confidence this will work for CAN